app/controllers/barbeque/sns_subscriptions_controller.rb

```diff
@@ -44,6 +44,6 @@ def destroy
   private
 
   def fetch_sns_topic_arns
-    Barbeque::SNSSubscriptionService.sns_client.list_topics.topics.map(&:topic_arn)
+    Barbeque::SNSSubscriptionService.sns_client.list_topics.flat_map{ |resp| resp.topics.map(&:topic_arn) }
```

Comment:
This is enought, isn't it.
`Barbeque::SNSSubscriptionService.sns_client.list_topics.flat_map(&:topics).map(&:topic_arn) }`

Reply:
Fixed. b28f812